web-react-ts/src/pages/campaigns/components/buttons/DefaulterDetailsCard.tsx

```diff
@@ -56,27 +56,43 @@ function DefaulterDetailsCard(props: DefaulterDetailsCardProps) {
           </div>
         )}
 
-        <div className="text-success">
+        <div
+          className={church?.fellowshipEquipmentFilledCount === 0 ? '' : 'good'}
```

Comment:
This one should also be checking whether this number is equal to the total number of fellowships